doc/INSTALL.md

```diff
@@ -1,38 +1,14 @@
-# Downloading LARS
-
-There are two ways that you can download LARS:
-
- * Downloading larsServer.zip from the Liberty Repository
- * By building larsServer.zip yourself
-
-Whichever download method you use, you will also need to configure
-LARS.
-
-## Downloading from the Liberty Repository
-
-Visit [this
-page](https://developer.ibm.com/wasdev/downloads/#asset/tools-Liberty_Asset_Repository_Service)
-and click Download to download `larsServer.zip`. 
-
-## By building and then unzipping larsServer.zip
-
-If you have built LARS yourself then you can find larsServer.zip in
-the `build/distributions` directory.
-
-# Installing and configuring LARS
+# Installing, configuring and running LARS
+LARS is a Java EE application running on top of Liberty and can be
+configured like any other Liberty application. When LARS is installed,
+it creates a new Liberty server called `larsServer`.
 
-Unzip larsServer.zip into the `WLP_USER_DIR` (usually `wlp/usr/`) directory of an existing Liberty 19.0.0.6 (or newer) runtime.
+Unzip `larsServer.zip` into the `wlp/usr/` directory of an existing Liberty 19.0.0.9 (or newer) runtime.
```

Comment:
I'm not sure why this change?  WLP_USER_DIR may not always be `wlp/usr`, and if it is configured otherly  then unzipping to `wlp/usr/` will not work